spec/support/fake_operations.rb

```diff
@@ -0,0 +1,15 @@
+module FakeOperations
+  extend self
```

Comment:
I really think `module_function` is more appropriate for this, but I'm sticking with `extend self` because RSpec is unable to call helper methods from within custom expectations if they are `module_functions`